v3.1/entrypoint.sh

```diff
@@ -1,5 +1,8 @@
 #!/bin/sh
 
+envsubst < /etc/clamav/clamd.conf.template > /etc/clamav/clamd.conf
+rm /etc/clamav/clamd.conf.template
```

Comment:
What's the point of removing the template? It's in the image anyway and in my opinion, it makes the process more obvious if this still exists in the running container.

Reply:
I see your point. I have removed the rm command from the script.